src/Shared/PropertyHelper/PropertyHelper.cs

```diff
@@ -151,6 +151,7 @@ public void SetValue(object instance, object? value)
     /// A cached array of all public properties of the specified type.
     /// </returns>
     [RequiresUnreferencedCode("This API is not trim safe.")]
+    [RequiresDynamicCode("This API is not trim safe.")]
```

Comment:
I looked at PropertyHelper.cs in my PR here - https://github.com/dotnet/aspnetcore/pull/45604/files#diff-8d64fa5ac96afa5f91aa693105c4f410335bfaa79ef790546ae1773e3f50f594

Instead of adding `RequiresDynamicCodeAttribute`, I removed dynamic code to avoid the attribute spreading everywhere.

Reply:
Thanks James!